README.md

```diff
@@ -40,22 +40,22 @@ The easiest way to manage multiple node versions in development is to use
 
 Here are some commands you can run:
 
-| Command              | Description                                                                     |
```

Comment:
This whitespace makes it hard to edit (and harder to read) the doc in an 80 char editor and doesn't add anything, so I removed it.